test/exercises/wavelet/wavelet.chpl

```diff
@@ -1,66 +1,78 @@
-use ecgIO;
-import Math.log2;
+use ecgIO, Math;
 
-config const nLevels = 3;
-config param plotResults = false;
+config const nLevels = 3,
+             plotResults = false;
+
+const SERIALIZE_THRESHOLD = 1024;
```

Comment:
Personally, all-caps-with-underscore constants look like C rather than Chapel to me.  I tend to think of capitals as being most useful for things that deserve to be distinguished (for me, classes, modules, and major arrays/domains) and don't think of this constant as being particularly significant or worthy of attention.  It also could make sense to support as a config so people could experiment with different thresholds.